File: src/main/scala/com/liyaos/metabenchmark/profiler/ArchiveDumper.scala
package com.liyaos.metabenchmark.profiler

/**
 * Created by lastland on 15/9/10.
 */
import java.io.PrintWriter

class ArchiveDumper(fileName: String) extends Dumper {
  val p = new PrintWriter(fileName)

  override def println(content: String) {
    p.println(content)
  }

  override def close() {
    p.close()
  }
}

File: src/main/java/com/liyaos/metabenchmark/profiler/ArchiveDumper.java
package com.liyaos.metabenchmark.profiler;

import java.io.File;
import java.io.FileNotFoundException;
import java.io.PrintWriter;

/**
 * Created by lastland on 15/10/6.
 */
public class ArchiveDumper implements Dumper {
    public PrintWriter p;

    public ArchiveDumper(String fileName) throws FileNotFoundException {
        p = new PrintWriter(fileName);
    }

    public void close() {
        p.close();
    }

    public void println(String s) {
        p.println(s);
    }
}
